### PR TITLE
Added MessageResponse DTO and added the ability to set response objects

### PR DESCRIPTION
**Pull request by JohnstonCode**
Related to https://github.com/messagebird/php-rest-api/pull/127

### CODE REVIEW

**Comment by JohnstonCode** [comment]
@CoolGoose do you have any issue with my implementation? If there aren't any i will look to add tests.

**Comment by CoolGoose** [comment]
Hey @JohnstonCode , thank you for the PR, this is looking good for me.

File: src/MessageBird/Objects/MessageResponse.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+     * @param mixed $object
+     *
+     * @return $this
+     */
+    public function loadFromArray($object)
+    {
+        parent::loadFromArray($object);
+
+        $this->recipients = (new Recipients())->loadFromArray($this->recipients);
+        $this->typeDetails = get_object_vars($this->typeDetails);
```
Comment (JohnstonCode) [inline src/MessageBird/Objects/MessageResponse.php:137]: The current implementation returns $typeDetails as a stdClass.

**Comment by CoolGoose** [comment]
Great, thank you 🙇

**Pull request merged**